AppConnectSwift/FormListViewController.swift

```diff
@@ -89,6 +94,100 @@ class FormListViewController: UITableViewController {
         tableView.reloadData()
     }
     
+    
+    func doSearch() {
+        
+        // the following data is sample fetch only
+        // used to illustrate how the appconnect 2.0 calls are supposed to work.
+        let clientFactory = MDClientFactory.sharedInstance()
+        let client = clientFactory?.client(of: MDClientType.network);
+
+        // take an arbitrary subjectUUID
+        let SubjectUUID = "686e525b-6608-46a3-bbb4-5079d97dcded";
+        
+        let formatter = DateFormatter()
+        formatter.dateFormat = "yyyy/MM/dd HH:mm";
+        let startDate = formatter.date(from: "2017/01/01 22:31");
+        let endDate = formatter.date(from: "2017/02/01 22:31");
+        
+        let parametersDictionary = ["size": "10", "page": "1"];
+        
+        // make the call to fetch all available records in this given time interval
+        // the results will be passed back as an NSDictionary
+        // in this stubbed method it will be ["SubjectUUID", "TestSubjectUUID"];
+        // when connected to a live server it will be something like 
+        // 
+        
+        //  subjectmetadatas = ["submission_collected_at": "2016-10-20T14:27:56.587Z",
+        // "content_type": "application/json",
+        // "file_size": "3456",
+        // "subject_uuid": "686e525b-6608-46a3-bbb4-5079d97dcded",...]
+        
+        client?.fetchAvailableSubjectMetadataWithSubjectUUID(withDateRange: SubjectUUID, from: startDate, to: endDate, withParameters: parametersDictionary) {
```

Comment:
Goodness, the automatic Swift bridge really messed up those method names. Might need a bit of massaging.

Also, methods from the Client interface that fetch data use the verb `load` instead of `fetch`. I have no preference for either but it would be good to be consistent and go with `load` here.